src/main/java/com/kosherjava/zmanim/hebrewcalendar/JewishCalendar.java

```diff
@@ -607,6 +605,9 @@ public int getYomTovIndex() {
 		final int day = getJewishDayOfMonth();
 		final int dayOfWeek = getDayOfWeek();
 
+		boolean fastOfEsther = ((day == 11 || day == 12) && dayOfWeek == Calendar.THURSDAY)
+				|| (day == 13 && !(dayOfWeek == Calendar.FRIDAY || dayOfWeek == Calendar.SATURDAY));
+
```

Comment:
This may not be what you want, but it removes the need to write the same code twice in both ADARs